core/blockchain.go

```diff
@@ -462,6 +463,16 @@ func NewBlockChain(db ethdb.Database, cacheConfig *CacheConfig, chainConfig *par
 	return bc, nil
 }
 
+func (bc *BlockChain) StartVerify(peers VerifyPeers, allowUntrustedVerify bool) {
```

Comment:
Can we not pass `allowUntrustedVerify` when `StartVerify`, but pass it when new the verifyManager.